.github/workflows/ql-for-ql-dataset_measure.yml

```diff
@@ -0,0 +1,80 @@
+name: Collect database stats for QL for QL
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - ql/ql/src/ql.dbscheme
+  pull_request:
+    branches: [main]
+    paths:
+      - ql/ql/src/ql.dbscheme
+  workflow_dispatch:
+
+jobs:
+  measure:
+    env:
+      CODEQL_THREADS: 4 # TODO: remove this once it's set by the CLI
+    strategy:
+      fail-fast: false
+      matrix:
+        repo: [github/codeql]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Fetch CodeQL
```

Comment:
See https://github.com/github/codeql/pull/7410/files#diff-c3c811f2f3a821590acf45fc281ba009b4886ec06d156f573019acf671e2d817R23 for how to obtain the codeql-cli that the codeql-action would be using. It is likely present on disk already, and it will by construction be the latest stable release.